src/main/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/Worker.java

```diff
@@ -88,9 +93,14 @@
 
     private static final Log LOG = LogFactory.getLog(Worker.class);
 
+    // Default configs for periodic shard sync
     private static final int SHARD_SYNC_SLEEP_FOR_PERIODIC_SHARD_SYNC = 0;
     private static final int MAX_INITIALIZATION_ATTEMPTS = 20;
     private static final int PERIODIC_SHARD_SYNC_MAX_WORKERS_DEFAULT = 1; //Default for KCL.
+    private static final long LEASE_TABLE_CHECK_FREQUENCY_MILLIS =1000;//= 3 * 1000L; MJ TODO
+    private static final long MIN_WAIT_TIME_FOR_LEASE_TABLE_CHECK_MILLIS = 0;//1 * 1000L;
+    private static final long MAX_WAIT_TIME_FOR_LEASE_TABLE_CHECK_MILLIS = 100;//30 * 1000L;
```

Comment:
Changed these while debugging unit tests, will change back to commented out values in next revision.

Reply:
For future reference, you can always use `git checkout -p` to partially stage changes

Reply:
Good to know!